es6-shim.js

```diff
@@ -1436,21 +1423,33 @@
     }
   };
   defineProperties(Math, MathShims);
-  // IE 11 TP has an imprecise log1p: reports Math.log1p(-1e-17) as 0
-  defineProperty(Math, 'log1p', MathShims.log1p, Math.log1p(-1e-17) !== -1e-17);
+  
+  var isNear = function (a, b) {
+    return Math.abs(1 - a / b) / Math.pow(2, -52) < 8;
```

Comment:
I put `Math.pow(2, -52)` specially, because our "expected" values are computed only for binary64, so, possibly, it is better to not use Number.EPSILON here.